joint_limits:

  right_wing_joint:
    has_position_limits: true
    min_position: 0
    max_position: 0.6
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0

  left_wing_joint:
   has_position_limits: true
   min_position: 0
   max_position: 0.6
   has_velocity_limits: false
   max_velocity: 0
   has_acceleration_limits: false
   max_acceleration: 0
   has_jerk_limits: false
   max_jerk: 0
   has_effort_limits: false
   max_effort: 0

  right_wheel_joint:
   has_position_limits: false
   min_position: -3.14
   max_position: 3.14
   has_velocity_limits: true
   max_velocity: 1
   has_acceleration_limits: false
   max_acceleration: 0
   has_jerk_limits: false
   max_jerk: 0
   has_effort_limits: false
   max_effort: 0

  left_wheel_joint:
   has_position_limits: false
   min_position: -3.14
   max_position: 3.14
   has_velocity_limits: true
   max_velocity: 1
   has_acceleration_limits: false
   max_acceleration: 0
   has_jerk_limits: false
   max_jerk: 0
   has_effort_limits: false
   max_effort: 0
